cmflib/cmf.py

```diff
@@ -132,6 +132,7 @@ def __init__(
         config.sqlite.filename_uri = filepath
         self.store = metadata_store.MetadataStore(config)
         self.filepath = filepath
+        self.dataslice_path = None
         self.child_context = None

```

Comment:
Why dataslice path is a cmf object property ?. One cmf object can have multiple executions and for each execution this path is different . So it is not safe to put it there

Reply:
removed